templates/shared/_case-study-allan-gray.html

```diff
@@ -0,0 +1,10 @@
+<section class="p-strip is-deep is-bordered">
+        <div class="row u-equal-height">
+          <div class="col-8">
+            <h2>Discover why Allan Gray turned to Canonical to boost efficiency and availability</h2>
+            <p>Allan Gray needed to ensure maximum availability for mission-critical systems, which is why it turned to Canonical for enterprise support for Kubernetes.</p>
+            <a class="p-button--neutral" href="/engage/k8s-allan-gray">Read the case study</a>
```

Comment:
The other case study include should also link to the engage page not "https://pages.ubuntu.com/Cloud_CS_ITstrategen.html"